### PR TITLE
Add org.lyx.LyX.Plugin.SweaveKnitr

### PR DESCRIPTION
**Pull request by bztd**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.lyx.LyX.Plugin.SweaveKnitr.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - type: file
+        url: https://cran.r-project.org/src/contrib/highr_0.10.tar.gz
+        sha256: ec55bc1ff66390ed66806dc2a7b6c17dbfd089b3d73fe2e369017f8cb4bc347b
+      - type: file
+        url: https://cran.r-project.org/src/contrib/xfun_0.41.tar.gz
+        sha256: 2374c3b8f141f1e5ff01b291d8aa5c3dc807dd8e99ab4dc7a048c9534a7e3b02
+      - type: file
+        url: https://cran.r-project.org/src/contrib/yaml_2.3.7.tar.gz
+        sha256: d20cb219e0f9c48aba02f132f81cfa9ecda5e22c925e36726840218ed56680ab
+  - name: gcc
```
Comment (bbhtt) [inline org.lyx.LyX.Plugin.SweaveKnitr.yml:47]: why do you need gcc?

**Comment by bztd** [comment]
bot, build org.lyx.LyX.Plugin.SweaveKnitr

File: org.lyx.LyX.Plugin.SweaveKnitr.yml (hunk 1 above)
Comment (hfiguiere) [reply org.lyx.LyX.Plugin.SweaveKnitr.yml:47]: one doesn't simply build gcc.

**Comment by bztd** [comment]
bot, build org.lyx.LyX.Plugin.SweaveKnitr

**Comment by bztd** [comment]
bot, build org.lyx.LyX.Plugin.SweaveKnitr

**Comment by bztd** [comment]
bot, build org.lyx.LyX.Plugin.SweaveKnitr

**Pull request closed without merging**